tests/factories/MonitorFactory.php

```diff
@@ -6,7 +6,7 @@
 
 $factory->define(Monitor::class, function (Faker\Generator $faker) {
     return [
-        'url' => 'http://localhost:9000',
+        'url' => sprintf('http://localhost:%d', getenv('TEST_SERVER_PORT')),
```

Comment:
When debugging I had to change the server port, and noticed this was hardcoded so I updated it to use the port from the phpunit env.

Reply:
Can we set `9000` as the default somewhere?

Reply:
The variable is set in the phpunit config here:
https://github.com/spatie/laravel-uptime-monitor/blob/master/phpunit.xml.dist

It's currently used here already:
https://github.com/spatie/laravel-uptime-monitor/blob/master/tests/Server.php#L63

We could set a constant as a default somewhere i guess, but since it's already set in the ENV i'm not sure it's needed.